dashboard/src/modules/components/OverviewComponent/index.jsx

```diff
@@ -84,8 +87,14 @@ const OverviewComponent = () => {
                 New and Unmanaged
               </AccordionToggle>
               <AccordionContent isHidden={!expanded.has("newRuns")}>
-                <NewRunsHeading />
-                <NewRunsComponent />
+                {newRuns.length > 0 ? (
+                  <>
+                    <NewRunsHeading />
+                    <NewRunsComponent />
+                  </>
```

Comment:
We didn't seem to need the empty tag at (new) line 91 before...so why do we need it now?  (I.e., what happens if we omit it?)

Reply:
Thanks, Varshini!